code/modules/clothing/shoes/jobs.dm

```diff
@@ -34,6 +34,9 @@
 	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
 	max_pressure_protection = FIRESUIT_MAX_PRESSURE
 	var/artificail_shine = 20
+	material = /decl/material/solid/metal/steel
```

Comment:
pls swap material and matter around. These actually use material for color, that'd result in iron shoes. And it's synthlether with steel insets, not steel ones.
so like
```
material = /decl/material/solid/leather/synth
matter = list(/decl/material/solid/metal/steel = MATTER_AMOUNT_REINFORCEMENT)
```